apis/v1alpha2/tlsroute_types.go

```diff
@@ -46,11 +46,23 @@ type TLSRoute struct {
 
 // TLSRouteSpec defines the desired state of a TLSRoute resource.
 type TLSRouteSpec struct {
-	// Gateways defines which Gateways can use this Route.
+	// ParentRefs references the resources that can attach to this Route. The
```

Comment:
Sorry if this is rehashing something in the doc, but this model feels backwards. IMO the mental model is routes attach to the gateway, not the other way around

Reply:
Yeah completely right, I'm not sure how this got out of sync but I cleaned this up to make more sense.